src/occurrence.jl

```diff
@@ -47,18 +46,47 @@ default page size GBIF uses). Future occurrences can be queried with `next!` or
 function occurrences(q::Dict)
   check_records_parameters!(q)
   occ_s_url = gbifurl * "occurrence/search"
-  occ_s_req = Requests.get(occ_s_url, query=q)
+  occ_s_req = HTTP.get(occ_s_url, query=q)
   if occ_s_req.status == 200
-    body = Requests.json(occ_s_req)
+    body = JSON.parse(String(occ_s_req.body))
     occ = map(GBIFRecord, body["results"])
     maxocc = body["count"] > 200000 ? 200000 : body["count"]
     return GBIFRecords(
       body["offset"],
       maxocc,
       q,
-      view(occ, collect(1:length(occ))),
       occ,
       ones(Bool, length(occ))
     )
   end
 end
+
+"""
+**Search for occurrences**
+
+Returns occurrences that correspond to a filter, given in `q` as a dictionary.
+When first called, this function will return the latest 20 hits (or whichever
+default page size GBIF uses). Future occurrences can be queried with `next!` or
+`complete!`.
+"""
```

Comment:
occurrences is also defined in SpatialEcology, will cause problems with interoperability

Reply:
Could this overload `Base.filter` instead?

Reply:
Is this still a sticking point? can I help?